data-processing/pulsar-flink/1.13/pulsar-flink.md

```diff
@@ -25,6 +25,10 @@ Pulsar Flink Connector is an integration of [Apache Pulsar](https://pulsar.apach
 
 ![](/images/data-processing/pulsar-flink-connector.png)
 
+> **Note**
+>
+> This legacy Pulsar Flink connector would not be maintained further. You can try the [new Pulsar Flink connector](../../pulsar-flink-new/1.15/1.15.0.0/pulsar-flink.md).
```

Comment:
For stronger wording, remove the modal verbs ("would", "can"): "This version of the Pulsar Flink connector is depreciated. For best performance and support, use the [new Pulsar Flink Connector]...."